db/post_migrate/20200330233427_drop_old_unread_pm_notification_indices.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class DropOldUnreadPmNotificationIndices < ActiveRecord::Migration[6.0]
+  def up
+    DB.exec("DROP INDEX IF EXISTS index_notifications_on_user_id_and_id")
+    DB.exec("DROP INDEX IF EXISTS index_notifications_on_read_or_n_type")
+  end
+
+  def down
+    add_index :notifications, [:user_id, :id], unique: true, where: 'notification_type = 6 AND NOT read'
+    execute <<~SQL
+      CREATE UNIQUE INDEX CONCURRENTLY index_notifications_on_read_or_n_type
```

Comment:
I don't think this will work cause it will still have a DDL transaction, just chuck a not allowed on down?